api/inbound-stream.js

```diff
@@ -50,7 +50,7 @@ class BridgeFn {
           let inst = {};
 
           inst.keys = _.map(y.keys, z => {
-            return z.toBase58();
+            return `${z.pubkey.toBase58()} (is_signer=${z.isSigner})`;
```

Comment:
Appending `is_signer` like this is a little hacky, but I wanted to show this to the user and wasn't prepared to dig through the UI to expose this properly as a separate field

Reply:
Fine for now, I think